src/internal/operations/test/exchange_test.go

```diff
@@ -514,7 +514,23 @@ func testExchangeContinuousBackups(suite *ExchangeBackupIntgSuite, toggles contr
 				require.NoError(t, err, clues.ToCore(err))
 
 				newLoc := expectDeets.MoveLocation(cat.String(), from.locRef, to.locRef)
+
+				// Remove ancestor folders of moved directory since they'll no longer
+				// appear in details since we're not backing up items in them.
+				pb, err := path.Builder{}.SplitUnescapeAppend(from.locRef)
+				require.NoError(t, err, "getting Builder for location: %s", clues.ToCore(err))
+
+				// Remove moved dir so we don't accidentally remove it.
+				pb = pb.Dir()
+
+				for len(pb.Elements()) > 0 {
```

Comment:
If we don't remove empty parent directories then we'll fail the test because we're doing exact comparisons on the details entries right now and corso hasn't yet been updated to preserve empty directories when making incremental backups

With the example you gave, the test should fail and require manual intervention prior to merging. I'm a bit less concerned about that since 1. these tests have been fairly stable and unchanged and 2. the test will fail and require attention rather than continuing to pass even if expectations aren't being met